cypress/integration/plugins/security-dashboards-plugin/inaccessible_tenancy_features.js

```diff
@@ -0,0 +1,25 @@
+/*
```

Comment:
@abhivka7 Where is the test setup for this test? Is it set to multi-tenancy disabled by default?

Reply:
That will be done through workflow in Security Dashboards plugin. Here is the commit: https://github.com/opensearch-project/security-dashboards-plugin/pull/1419/commits/b70c248a32e8b235a0131ccafb5cd11bf178a972 

PR: https://github.com/opensearch-project/security-dashboards-plugin/pull/1419/commits